src/html/classic/ng/src/web/pages/extras/trashcanpage.js

```diff
@@ -0,0 +1,493 @@
+/* Greenbone Security Assistant
+*
+* Authors:
+* Steffen Waterkamp <steffen.waterkamp@greenbone.net>
+*
+* Copyright:
+* Copyright (C) 2017 Greenbone Networks GmbH
+*
+* This program is free software; you can redistribute it and/or
+* modify it under the terms of the GNU General Public License
+* as published by the Free Software Foundation; either version 2
+* of the License, or (at your option) any later version.
+*
+* This program is distributed in the hope that it will be useful,
+* but WITHOUT ANY WARRANTY; without even the implied warranty of
+* MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+* GNU General Public License for more details.
+*
+* You should have received a copy of the GNU General Public License
+* along with this program; if not, write to the Free Software
+* Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+*/
+
+import React from 'react';
+
+import _ from 'gmp/locale.js';
+
+import TrashActions from './trashactions.js';
```

Comment:
Please move the import to the bottom as the last import. Generic pattern for imports is currently:
first react, afterwards external components, then gmp imports then web imports, last imports from same current dir.

Reply:
Resorting done